src/test/java/com/purbon/kafka/topology/integration/SaslPlaintextKafkaContainer.java

```diff
@@ -0,0 +1,142 @@
+package com.purbon.kafka.topology.integration;
+
+import com.github.dockerjava.api.command.ExecCreateCmdResponse;
+import com.github.dockerjava.api.command.InspectContainerResponse;
+import java.nio.charset.StandardCharsets;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.concurrent.TimeUnit;
+import org.apache.kafka.clients.CommonClientConfigs;
+import org.apache.kafka.clients.admin.Admin;
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.Transferable;
+import org.testcontainers.utility.DockerImageName;
+
+/** @author <a href="mailto:shh@thathost.com">Sverre H. Huseby</a> */
```

Comment:
Can you remove this comment? authorship is going to be for sure keep as you are the creator and committer of the file.

Reply:
Sure! It's part of my IDE class template. :-)

Reply:
Thanks ! appreciate, again thanks a ton for this, is a very valuable contribution